themes/cclw/constants/faqs.tsx

```diff
@@ -502,7 +502,7 @@ export const FAQS: TFAQ[] = [
         the NewClimate Institute and its partners for the{" "}
         <ExternalLink url="https://climatepolicydatabase.org/">Climate Policy Database</ExternalLink> project. Prior to their ingest into the
         database, documents were reviewed by researchers at Climate Policy Radar to ensure that no duplicate entries were added and that entries fell
-        within the scope of the Climate Change Laws of the World  <ExternalLink url="https://climatepolicydatabase.org/">methodology</ExternalLink>.
+        within the scope of the Climate Change Laws of the World <ExternalLink url="https://climate-laws.org/methodology">methodology</ExternalLink>.
         This data has been ingested with permission from NewClimate Institute, in order to advance efforts by Climate Policy Radar and LSE to provide
```

Comment:
Updating this per 
![Screenshot 2024-08-19 at 16 52 39](https://github.com/user-attachments/assets/66013b1e-69ed-477e-9b7d-12be4309bb5a)